vms/evm/destination_client.go

```diff
@@ -434,22 +433,26 @@ func (s *concurrentSigner) issueTransaction(
 	sendTxCtx, sendTxCtxCancel := context.WithTimeout(context.Background(), utils.DefaultRPCTimeout)
 	defer sendTxCtxCancel()
 
-	s.logger.Info(
+	log := s.logger.With(
+		zap.Stringer("txID", signedTx.Hash()))
```

Comment:
We might as well add the nonce and the gas values to this `logger.With` for the rest of the log calls in this function